src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/RequestBookingService.kt

```diff
@@ -43,12 +43,21 @@ class RequestBookingService(
   private val referenceCodeRepository: ReferenceCodeRepository,
   private val notificationRepository: NotificationRepository,
   private val locationsInsidePrisonClient: LocationsInsidePrisonClient,
+  private val availabilityService: AvailabilityService,
 ) {
   companion object {
     private val log = LoggerFactory.getLogger(this::class.java)
   }
 
-  fun request(request: RequestVideoBookingRequest, user: User) {
+  fun request(request: RequestVideoBookingRequest, user: ExternalUser) {
```

Comment:
Only external users should be using this service.

Reply:
There was no availability check in the request a booking journey before? Does it need one now? I think this just sends an email to the prison to book and confirm it. It wouldn't have failed the request for this reason before.

Reply:
I checked with Steve on this one.  There is an availability check done in the UI for request a booking.